test/escalus_stanza_SUITE.erl

```diff
@@ -53,7 +53,7 @@ attribute_as_argument(_) ->
     Attr = {<<"name">>, <<"value">>},
     Example = #xmlel{name = <<"el">>,
                      attrs = [Attr]},
-    ?eq(Example, M:from_template("<el {{attr}}/>", [{attr, Attr}])).
+    ?eq(Example, M:from_template("<el {{{attr}}}/>", [{attr, Attr}])).
```

Comment:
What exactly does this change do? We might need to update tests when switching to this new version if this is required.